src/components/SupportRequest.js

```diff
@@ -24,248 +24,217 @@ const constraints = {
-              'Drag or', h(Link, { style: { margin: '0 0.25rem' } }, ['Click']), 'to attach a file',
-              icon('upload-cloud', { size: 25, style: { opacity: 0.4, marginLeft: '0.5rem' } })
+            div({
+              style: { marginLeft: '1rem', paddingTop: '0.5rem' }
+            }, [
+              'Successfully uploaded: ', span({ style: { color: colors.dark() } }, [attachmentName])
             ])
           ]),
-        uploadingFile && spinnerOverlay,
-        h(IdContainer, [id => h(Fragment, [
-          h(FormLabel, { required: true, htmlFor: id }, ['Contact email']),
-          h(TextInput, {
-            id,
-            value: email,
-            placeholder: 'Enter your email address',
-            onChange: v => this.setState({ email: v })
-          })
-        ])]),
-        h(FormLabel, { required: true }, ['Are you a clinical user?']),
-        h(RadioButton, {
-          text: 'Yes', name: 'is-clinical-user', checked: clinicalUser === true,
-          labelStyle: { margin: '0 2rem 0 0.25rem' },
-          onChange: () => this.setState({ clinicalUser: true })
-        }),
-        h(RadioButton, {
-          text: 'No', name: 'is-clinical-user', checked: clinicalUser === false,
-          labelStyle: { margin: '0 2rem 0 0.25rem' },
-          onChange: () => this.setState({ clinicalUser: false })
-        }),
-        submitError && div({ style: { marginTop: '0.5rem', textAlign: 'right', color: colors.danger() } }, [submitError]),
-        submitting && spinnerOverlay,
-        div({ style: styles.buttonRow }, [
-          h(ButtonSecondary, {
-            style: { marginRight: '1rem' },
-            onClick: () => SupportRequest.dismiss()
-          }, ['Cancel']),
-          h(ButtonPrimary, {
-            disabled: errors,
-            tooltip: Utils.summarizeErrors(errors),
-            onClick: () => this.submit()
-          }, ['SEND'])
-        ])
-      ])])
-    ])
-  }
-
-  static dismiss() {
-    contactUsActive.set(false)
-  }
+          h(Link, {
+            tooltip: 'Remove file',
+            style: { flex: 0, paddingTop: '0.5rem' },
+            onClick: () => {
+              setAttachmentToken('')
+              setAttachmentName('')
+            }
+          }, [icon('times-circle', { size: 23 })])
+        ]) :
+        h(Clickable, {
+          style: {
+            flex: 1, backgroundColor: dragging ? colors.accent(0.2) : colors.dark(0.1), borderRadius: 3,
+            border: `1px dashed ${colors.dark(0.7)}`
+          },
+          onClick: openUploader
+        }, [
+          div({ style: { fontSize: 14, lineHeight: '30px', paddingLeft: '1rem', display: 'flex', alignItems: 'center' } }, [
+            'Drag or', h(Link, { style: { margin: '0 0.25rem' } }, ['Click']), 'to attach a file',
+            icon('upload-cloud', { size: 25, style: { opacity: 0.4, marginLeft: '0.5rem' } })
+          ])
+        ]),
+      uploadingFile && spinnerOverlay,
```

Comment:
@ehanna4 and I were testing this and we noticed that the spinner is not justified correctly in the modal. I don't think it came from this PR but might be worth fixing because it doesn't look very nice.